exasol_integration_test_docker_environment/version.py

```diff
@@ -1,9 +1,13 @@
-# ATTENTION:
-# This file is generated by exasol/toolbox/nox/_package_version.py when using:
-#   * either "poetry run -- nox -s project:fix"
-#   * or "poetry run -- nox version:check -- --fix"
-# Do not edit this file manually!
-# If you need to change the version, do so in the pyproject.toml, e.g. by using `poetry version X.Y.Z`.
+"""
+ATTENTION:
+    This file is generated by exasol/toolbox/nox/_package_version.py when using:
+        * either "poetry run -- nox -s project:fix"
+        * or "poetry run -- nox -s version:check -- --fix"
+Do not edit this file manually!
+If you need to change the version, do so in the pyproject.toml, e.g. by using
+`poetry version X.Y.Z`.
+"""
+
```

Comment:
Not exactly the file created by `nox -s project:fix`, because of https://github.com/exasol/python-toolbox/issues/514